labelbox/schema/identifiable.py

```diff
@@ -0,0 +1,65 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Union
+
+
+class IdType(str, Enum):
+    DataRowId = "UID"
+    GlobalKey = "GLOBAL_KEY"
+
+
+class Identifiable(ABC):
+
+    def __init__(self, keys: Union[str, List[str]], id_type: IdType):
+        self._keys = keys
+        if isinstance(keys, str):
+            self._keys = [keys]
+        self._id_type = id_type
+
+    @property
+    def keys(self):
+        return self._keys
+
+    @keys.setter
+    def keys(self, keys):
+        self._keys = keys
+        if isinstance(keys, str):
+            self._keys = [keys]
+
+    @classmethod
+    @abstractmethod
+    def strings_to_identifiable(cls, keys: Union[str, List[str]]):
+        pass
+
+    def __eq__(self, other):
+        return other.keys == self.keys
+
+    def __hash__(self):
+        hash(self.keys)
+
+    def __str__(self):
+        return self.keys.__str__()
+
+
+class UniqueIds(Identifiable):
+
+    @classmethod
+    def strings_to_identifiable(cls, keys: Union[str, List[str]]):
+        return cls(keys)
+
+    def __init__(self, keys: Union[str, List[str]]):
+        super().__init__(keys, IdType.DataRowId)
+
+
+class GlobalKeys(Identifiable):
+
+    @classmethod
+    def strings_to_identifiable(cls, keys: Union[str, List[str]]):
+        return cls(keys)
+
+    def __init__(self, keys: Union[str, List[str]]):
+        super().__init__(keys, IdType.GlobalKey)
+
+
+DefaultIdentifiable = UniqueIds
+DataRowIdentifiers = Union[UniqueIds, GlobalKeys]
```

Comment:
missing a newline at the end of the file

Reply:
From a user point of view, it does not seem clear that UniqueIds is DataRowIds, since global keys are also unique and they are also ids. What do you think of creating a sub-class / alias for `UniqueIds` called `DataRowIds`

Reply:
Hi @whistler I am trying to avoid 'over-classing' but I see your point. Would something like this help
`DataRowIds = UniqueIds` i.e. kind of class alias
PS We will also document this in a number of places

Reply:
Confused why this code even exists really.

Reply:
Yes @vbrodsky, that seems good to me.